src/context/WalletProvider/KeepKey/KeepKeyTypes.ts

```diff
@@ -93,6 +93,8 @@ export enum MessageType {
   RESETDEVICE = 14,
   SIGNTX = 15,
   FEATURES = 17,
+  PINREQUEST = 'PIN_REQUEST',
+  PINCHANGED = 'PIN changed',
```

Comment:
These are _messages_, not _message types_. We need to either use the numerical code with these message types, or put these into a new `Message` enum and use that.

The former is preferred.